clang/test/CXX/drs/dr5xx.cpp

```diff
@@ -265,6 +269,18 @@ namespace dr527 { // dr527: na
   int ax = a.x, bx = b.x, cx = c.x, dx = d.x, ex = E::e->x, fx = F::f->x;
 }
 
+namespace dr528 { // dr528: 2.7
```

Comment:
This is NAD, should we not say that?

Reply:
For NAD issues, we write tests that check for status quo, and report the first version where we exhibit the behavior CWG wants.